src/OnboardingSPA/components/LivePreview/BlockPreview/index.js

```diff
@@ -53,7 +62,16 @@ const BlockPreview = ( {
 	}, [ storedPreviewSettings ] );
 
 	return (
-		<div className={ `live-preview__container-${ styling }` }>
+		<div className={`live-preview__container-${styling }  live-preview-container` }>
+			{isShown ? null :
+				<div className='is-skeleton'>
```

Comment:
```suggestion
				<div className='live-preview__container-${styling }--is-skeleton'>
```
And prefix `__` and `--` whenever necessary in the new divs as well https://getbem.com/naming/

Reply:
Sure going for live-preview__container--is-skeleton cause else the code will be duplicated in both styles